search_events_app/views.py

```diff
@@ -1,10 +1,13 @@
 from django.shortcuts import render
 from django.views.generic.list import ListView
 
+from .services.dummy_api import DummyApi
+
 
 # Create your views here.
 class EventListView(ListView):
     template_name = 'event_list.html'
 
     def get_queryset(self):
-        return None
+        d = DummyApi()
+        return d.get()
```

Comment:
It's possible to return DummyApi().get()?. It does not possible, could you choose a more specific variable name?

Reply:
Changed to DummyApi().get() :)